js/filters.js

```diff
@@ -33,15 +33,24 @@ function Filter(elm) {
   this.$body = $(elm);
   this.$input = this.$body.find('input:not([name^="_"])');
   this.$remove = this.$body.find('.button--remove');
+  this.$inputFilter = this.$body.find('.input--removable input');
```

Comment:
Maybe a topic for a separate PR, but I wonder if we should break out the text input filters from the checkbox filters into their own classes, so that we don't have as much logic here?